docs/pages/includes/plugins/install-access-request.mdx

```diff
@@ -0,0 +1,63 @@
+<Tabs>
+<TabItem label="Download">
+
+We currently only provide `linux-amd64` binaries. You can also compile these
+plugins from source. You can run the plugin from a remote host or your local
+development machine.
+
+  ```code
+  $ curl -L -O https://get.gravitational.com/teleport-access-{{ name }}-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
+  $ tar -xzf teleport-access-{{ name }}-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
+  $ cd teleport-access-{{ name }}
+  $ ./install
+  ```
+
+  Make sure the binary is installed:
+
+  ```code
+  $ teleport-{{ name }} version
+  teleport-{{ name }} v(=teleport.plugin.version=) git:teleport-{{ name }}-v(=teleport.plugin.version=)-fffffffff go(=teleport.golang=)
+  ```
+
+</TabItem>
+<TabItem label="From Source">
+  To install from source you need `git` and `go` installed. If you do not have Go installed, visit the Go [downloads page](https://go.dev/dl/).
+
+  ```code
+  # Checkout teleport-plugins
+  $ git clone https://github.com/gravitational/teleport-plugins.git
+  $ cd teleport-plugins/access/{{ name }}
+  $ make
+  ```
+
+  Run `./install` from `teleport-{{ name }}` or place the executable in the
```

Comment:
I opened an issue in the plugins repo when testing this and found that the `install` script assumes the binary is in the CWD. I was [told](https://github.com/gravitational/teleport-plugins/pull/840#issuecomment-1570436183) that this script is to be used with the packaged tarballs, so this needs to be adjusted.